packages/layout/src/steps/getFootnotePlaceholder.js

```diff
@@ -0,0 +1,16 @@
+function getFootnotePlaceholder(node) {
```

Comment:
I feel like this does not fit in as a file in `steps` directory. Rather inside the `footnotes` directory (since it is not a step fed into the `index.js` file.